src/metpy/__init__.py

```diff
@@ -28,6 +29,7 @@
 
 # Must occur before below imports
 warnings.filterwarnings('ignore', 'numpy.dtype size changed')
+os.environ['PINT_ARRAY_PROTOCOL_FALLBACK'] = '0'
```

Comment:
Is there a reason this is here and not in `units.py`?

Reply:
If it is here, then we don't need the `noqa` on the import order break in `units.py`, but I can easily change it to that if it is preferred.

Reply:
Ok, that's fine. Let's leave it.